docs/vignettes/vignette1/configs/populate_configs.sh

```diff
@@ -0,0 +1,20 @@
+#! /usr/bin/env bash
```

Comment:
Is this file supposed to be here?

I also think I can delete `docs/vignettes/vignette1/configs/clean_configs.sh`

Reply:
That can go away. I thought we had a convenience script for "resetting" but I guess not.